src/index.js

```diff
@@ -1,6 +1,9 @@
 import { months } from './data.js'
+
```

Comment:
You can try exploring on es6+ syntax if that can be used, like arrow functions

Reply:
I will resolve it in next PR, Sai